CONTRIBUTING.md

```diff
@@ -150,6 +150,27 @@ We're currently experimenting with hosting the majority of lodestar packages and
   - Error: One or more main functionalities are not working, preventing some functions from working properly.
   - Fatal: One or more main functionalities are not working and preventing the application from fulfilling its duties.
 
+## Logging policy
+
+### Logging Levels
+
+- `error`: Critical issues that prevent the application from functioning correctly or cause significant disruption to users. Examples include failed network connections, crashes, or data corruption.
+- `warn`: Situations that may lead to critical issues if not addressed but do not prevent the application from functioning. Examples include configuration issues, deprecated features, or temporary network disruptions.
+- `info`: General sporadic informational about the node's state. Examples include initialization messages, infrequent periodic status updates, or high-level progress reports.
+- `verbose`: _TBD_
+- `debug`: Detailed diagnostic information that can help developers or users troubleshoot specific issues. Examples include individual request logs for every REST API, networking interactions, or internal components status changes.
+
+### Guideliness
```

Comment:
Are we ok with not adding a bit more details about stack traces? I think the `ApiError` is a pretty good example of when not to include the stack trace as changed in https://github.com/ChainSafe/lodestar/pull/5285

- it is thrown on a higher level in the stack
- it is explicitly thrown based on simple assertions
- the error is expected to happen due to invalid client requests
- the error message is usually good which makes it easy to find out where it was thrown

Not including the stack trace of course can come with a cost of not being able to easily debug an issue but this should be rarely the case if we are careful to only do it for some errors but let's not forget the cost of including the stack trace in terms of UX. Not being able to easily read the logs because of too much stack trace is also not ideal.

I would argue that `debug` logs can include stack traces more often/always but for levels higher than or equal `info` need to be more careful.

Reply:
I just approved the PR so this didn't sit for too long. Happy to merge more verbiage about when stack traces are acceptable. Totally agree with your comment.